cmd/crc/cmd/oc_env.go

```diff
@@ -33,7 +35,7 @@ var ocEnvCmd = &cobra.Command{
 			output.Outln(shell.GetEnvString(userShell, "HTTPS_PROXY", proxyConfig.HttpsProxy))
 			output.Outln(shell.GetEnvString(userShell, "NO_PROXY", proxyConfig.GetNoProxyString()))
 		}
-		output.Outln(shell.GenerateUsageHint(userShell, "crc oc-env"))
+		output.Outln(shell.GenerateUsageHint(userShell, os.Args[0]+" oc-env"))
```

Comment:
`os.Args[0]` needs to be wrapped in \' \' in case it contains spaces, semi colon, ...